Moving parts of the Shape Registry into a common file

Gamrix opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67575
* #67500
* #66973
* __->__ #66948

Differential Revision: [D32497550](https://our.internmc.facebook.com/intern/diff/D32497550)

Gamrix committed:
Moving parts of the Shape Registry into a common file

[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Krovatkin reviewed (approved):
:shipit:

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Krovatkin commented on torch/csrc/jit/passes/utils/op_registry.cpp:
```diff
@@ -0,0 +1,72 @@
+      "aten::upsample_trilinear3d(Tensor self, int[] output_size, bool align_corners, float? scales_d, float? scales_h, float? scales_w) -> Tensor",
+      "aten::prelu(Tensor self, Tensor weight) -> Tensor",
+  });
+  return ops;
```
I wonder why they didn't use a plain reference instead, but it isn't something you changed or introduced.

Krovatkin reviewed (approved):
:shipit:

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"




[ghstack-poisoned]

Gamrix commented:
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32175960).

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"


Differential Revision: [D32175960](https://our.internmc.facebook.com/intern/diff/D32175960)

[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"


Differential Revision: [D32175960](https://our.internmc.facebook.com/intern/diff/D32175960)

[ghstack-poisoned]

Gamrix commented:
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32175960).

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"


Differential Revision: [D32175960](https://our.internmc.facebook.com/intern/diff/D32175960)

[ghstack-poisoned]

Gamrix committed:
Update on "Moving parts of the Shape Registry into a common file"


Differential Revision: [D32175960](https://our.internmc.facebook.com/intern/diff/D32175960)

[ghstack-poisoned]

Gamrix commented:
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32497550).
